ERCS/erc-7704.md

```diff
@@ -0,0 +1,172 @@
+---
+eip: 7704
+title: Soul Resonance Token
+description: this EIP defines an interface for Soul Resonance Tokens
```

Comment:
Your description adds no new information that wasn't in the title. The description should elaborate on the ideas introduced in the title, not just repeat them. This would be a good place to define what a soul resonance token is.